test/functional/README.md

```diff
@@ -1,3 +1,3 @@
 # Kibana Functional Testing
 
-See our [Functional Testing Guide](https://www.elastic.co/guide/en/kibana/current/development-functional-tests.html)
+See our [functional testing README.md](https://github.com/elastic/kibana/blob/master/x-pack/README.md#running-functional-tests) for more information.
```

Comment:
The docs this previously linked to just seem to have moved slightly. Do we want to keep linking to the official docs?

```suggestion
See our [Functional Testing Guide](https://www.elastic.co/guide/en/kibana/current/development-tests.html#development-functional-tests) for more information.
```

Reply:
If the links don't get redirected when they change, this creates a pretty frustrating experience. (I didn't know they moved, looked like they were just gone.)